atlassian_jwt_auth/contrib/flask_app/decorators.py

```diff
@@ -0,0 +1,30 @@
+from functools import wraps
+
+from flask import Response, current_app, request
+
+import atlassian_jwt_auth
+from atlassian_jwt_auth.contrib.flask_app.utils import parse_jwt
+
+
+def requires_asap(f):
+    """
+    Wrapper for Flask endpoints to make them require authentication to access.
+    """
+
+    @wraps(f)
+    def decorated(*args, **kwargs):
+        verifier = _get_verifier()
+        auth = request.headers.get('Authorization')
+        if not auth or not parse_jwt(verifier, auth.split(" ")[1]):
```

Comment:
I'd be careful around `auth.split(" ")[1]`  <- what if the header has an empty value (can we hit an IndexError) ?